DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -279,8 +280,47 @@ final class SRSnapshotTests: SnapshotTestCase {
 
     func testNavigationBars() throws {
 
+        let privacyModes: [SessionReplayPrivacyLevel] = [.allow, .mask]
+
+        // - Static Navigation Bars -
+
+        // Note: Static Navigation Bars are not representative of realistic rendering at runtime.
+        // Therefore, Embedded Navigation Bars snapshot tests are also included for more accurate simulations.
+
+        // Test Static Navigation Bars without tinted color
+        show(fixture: .navigationBars)
+        try forPrivacyModes(privacyModes) { privacyMode in
+            let image = try takeSnapshot(with: privacyMode)
+            let fileNamePrefix = Fixture.navigationBars.slug
+            DDAssertSnapshotTest(
+                newImage: image,
+                snapshotLocation: .folder(named: snapshotsFolderPath, fileNameSuffix: "-\(fileNamePrefix)-\(privacyMode)-privacy"),
+                record: recordingMode
+            )
+        }
+
+        // Test Static Navigation Bars with tinted color
+        let vc2 = show(fixture: .navigationBars) as! NavigationBarControllers
+        vc2.setTintColor()
+        // Wait for colors to appear on screen, so the recorder can capture them.
+        wait(seconds: 0.5)
```

Comment:
question/ This might become flaky. Is there an option to assure that colors are set upon `setTintColor()` return? Might be tricky 🤔 💭

Reply:
I can create another view controller in the storyboard with all the tint colors set, then no need to call a specific method or wait for half a sec.

Reply:
It feels not worth it and I don't see any other option. Let's maybe check with x100 reps (in local) that `0.5s` delay isn't flaky and we're good to go 👍.